src/pages/EditMaintenance.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+    <transition name="slide-fade" appear>
+        <div>
+            <h1 class="mb-3">{{ pageName }}</h1>
+            <form @submit.prevent="submit">
+                <div class="shadow-box">
+                    <div class="row">
+                        <div class="col-md-6">
+                            <h2 class="mb-2">{{ $t("General") }}</h2>
+
+                            <!-- Title -->
+                            <div class="my-3">
+                                <label for="name" class="form-label">{{ $t("Title") }}</label>
+                                <input id="name" v-model="maintenance.title" type="text" class="form-control"
+                                       :placeholder="titlePlaceholder" required>
+                            </div>
+
+                            <!-- Description -->
+                            <div class="my-3">
+                                <label for="description" class="form-label">{{ $t("Description") }}</label>
+                                <textarea id="description" v-model="maintenance.description" class="form-control"
+                                          :placeholder="descriptionPlaceholder"></textarea>
+                            </div>
+
+                            <!-- Affected Monitors -->
+                            <div class="my-3">
+                                <label for="affected_monitors" class="form-label">{{ $t("Affected Monitors") }}</label>
+
+                                <VueMultiselect
+                                    id="affected_monitors"
+                                    v-model="affectedMonitors"
+                                    :options="affectedMonitorsOptions"
+                                    track-by="id"
+                                    label="name"
+                                    :multiple="true"
+                                    :allow-empty="false"
+                                    :close-on-select="false"
+                                    :clear-on-select="false"
+                                    :preserve-search="true"
+                                    :placeholder="$t('Pick Affected Monitors...')"
+                                    :preselect-first="false"
+                                    :max-height="600"
+                                    :taggable="false"
+                                ></VueMultiselect>
+
+                                <div class="form-text">
+                                    {{ $t("affectedMonitorsDescription") }}
+                                </div>
+                            </div>
+
+                            <!-- Start Date Time -->
+                            <div class="my-3">
+                                <label for="start_date" class="form-label">{{ $t("Start of maintenance") }}</label>
```

Comment:
It will be good to add current timezone in label?

Reply:
Did you mean this way?

![image](https://user-images.githubusercontent.com/59953812/150690655-cb06d7ff-28f0-4e42-8f57-ab7ffc56530b.png)


I see, for example, that heartbeats are stored in UTC format. Should I convert user input to UTC format and use the "old" `datetime()` function?

Reply:
No, in label, not in input...

For example:
```
Start of Maintenance (GMT+1)
<input ...>
```

Reply:
I understand. I will now commit with minor fixes (missing commas, spaces, translations) and try to make some adjustments to storing maintenance dates in the database.

Currently, the date is stored in the database in the time zone in which the date was entered by the user, not in UTC (+0) format, so it cannot be converted to the current time zone, eg when changing the time zone.
Ie. if the user enters 1:35 while creating maintenance and changes the time zone, it will still show 1:35.